arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.stm32f103x8

```diff
@@ -1,19 +1,18 @@
 # Kconfig - ST Microelectronics STM32F103RB MCU
 #
```

Comment:
why are make changes and renaming this file, if we are unifying things in Kconfig.defconfig.stm32f103xx